File: src/main/kotlin/xingkong/loopicserver/picproc/RenameRandDir.java
package xingkong.loopicserver.picproc;


import xingkong.loopicserver.picproc.multiDir.SubDirImages;

import java.util.Scanner;

public class RenameRandDir {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("------------------------------------");
        in.useDelimiter("\n");
        String from = in.next();
        String to = in.next();
        SubDirImages sb = new SubDirImages();
        sb.copyRandImage(from, to, 30);

    }
}

File: src/main/kotlin/xingkong/loopicserver/picproc/RenameRandDir.kt
package xingkong.loopicserver.picproc


import xingkong.loopicserver.picproc.multiDir.SubDirImages
import java.util.*

object RenameRandDir {

    @JvmStatic
    fun main(args: Array<String>) {
        val `in` = Scanner(System.`in`)
        println("------------------------------------")
        `in`.useDelimiter("\n")
        val from = `in`.next()
        val to = `in`.next()
        val sb = SubDirImages()
        sb.copyRandImage(from, to, 30)

    }
}
